docs/can-fixture.md

```diff
@@ -132,7 +132,7 @@ var todoStore = fixture.store([
   { id: 2, name: 'Walk the dog'}
 ], todoAlgebra);
 
-fixture("/api/todos/{id}", todoStore);
+fixture("/api/todos/{id}", todoStore); // can also be written fixture("/api/todos", todoStore);
```

Comment:
can we show the `fixture("/api/todos")` way and the old way will be the "also be written".

I'd like to standardize on the newer syntax.